src/Altinn.ApiClients.Maskinporten/Service/MaskinportenService.cs

```diff
@@ -0,0 +1,155 @@
+﻿using Altinn.ApiClients.Maskinporten.Config;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Altinn.ApiClients.Maskinporten.Services
+{
+    public class MaskinportenService: IMaskinporten
+    {
+        private readonly HttpClient _client;
+
+        private readonly MaskinportenSettings _maskinportenConfig;
+
+        public MaskinportenService(HttpClient httpClient, IOptions<MaskinportenSettings> maskinportenConfig)
+        {
+            httpClient.BaseAddress = new Uri("https://ver2.maskinporten.no");
+            httpClient.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+            httpClient.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/xml"));
```

Comment:
This will tell Maskinporten that it can respond with XML. Not sure we want it to do that.

Reply:
fixed